src/views/Comment/CommentGroup.js

```diff
@@ -57,6 +59,9 @@ CommentGroup.propTypes = {
   /** Comments can hide extra information unless a user shows intent to interact with a comment. */
   minimal: PropTypes.bool,
 
+  /** Comments can have different sizes*/
+  size: PropTypes.oneOf(['mini', 'tiny', 'small', 'large', 'big', 'huge', 'massive']),
```

Comment:
We use `SIZES` const for definitions, [example](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/collections/Message/Message.js#L94).